resteasy-core/src/main/java/org/jboss/resteasy/core/ExceptionHandler.java

```diff
@@ -103,7 +103,7 @@ protected Response handleApplicationException(HttpRequest request, ApplicationEx
          return jaxrsResponse;
       }
       jaxrsResponse = unwrapException(request, e, logger);
-      if (jaxrsResponse == null) {
+      if (jaxrsResponse == null && providerFactory.isDefaultExceptionManagerEnabled()) {
          jaxrsResponse = providerFactory.getThrowableExceptionMapper().toResponse(e.getCause());

```

Comment:
Do I misunderstand or are you disabling all the exception mappers if we disable the default one by doing that?

It doesn't seem like the behavior we want. We want to disable the default one but also allow the users to install an exception mapper.

Reply:
That's a good catch. It will break if the user disables the default exception mapper, but adds their own `ExceptionMapper<Throwable>`.

Reply:
This actually does work and can be kept like this. I'll add some comments for clarify in the source though.